posthog/async_migrations/setup.py

```diff
@@ -33,18 +38,51 @@ def reload_migration_definitions():
 reload_migration_definitions()
 
 
+def is_fresh_install(first_migration: str):
```

Comment:
Nit: File ordering became weird here. We generally follow the principle of more important stuff first, less important last